main/BUILD

```diff
@@ -4,12 +4,38 @@ load("@deps//:requirements.bzl", "requirement")
 py_binary(
     name = "train_classifier_g",
     srcs = ["train_classifier_g.py"],
+    visibility = ["//visibility:public"],
     deps = [
-        "//src:receive_data",
+        "//src:classifiers",
         "//src:download_data",
+        "//src:receive_data",
         "//src:suite",
-        "//src:classifiers",
-        requirement("pandas")
+        requirement("pandas"),
     ],
 )
 
+load(
+    "@com_grail_rules_r//R:defs.bzl",
+    "r_binary",
+    "r_library",
+    "r_pkg_test",
+    "r_unit_test",
+)
+load("@com_grail_rules_r//R:defs.bzl", "r_package")
+load("@com_grail_rules_r//R:defs.bzl", "r_package_with_test")
+load("@r_repositories_bzl//:r_repositories.bzl", "r_repositories")
+
+r_binary(
+    name = "feed_data",
+    src = "feed_data.R",
+    data = [
```

Comment:
I'm surprised that these data dependencies have to be defined twice, both here and in the other BUILD file. also, this one  has 4 files, the other one has 5. can you reduce the duplication?

Reply:
I think one is to export from `src`, one is to read in to `main`. I took it out and then bazel complained.